build_docs_main failing: Duplicates in PYPI_PACKAGES and installer packages: ['mne-kit-gui']

The docs build is asserting during sphinx extension setup:

```
File "doc/sphinxext/related_software.py", line 124, in _get_packages
AssertionError: Duplicates in PYPI_PACKAGES and installer packages: ['mne-kit-gui']
```

`mne-kit-gui` is listed in `PYPI_PACKAGES` (with the comment *"moved to its own env in the installers"*), but it still shows up in the installer manifest fetched by `_get_installer_packages()`, so the disjointness assertion fails.

It's intermittent — on the same commit (latest main, `1a442f0c6`), `build_docs` passed but `build_docs_main` failed, depending on whether the manifest fetch succeeds. The fix is probably to drop `mne-kit-gui` from `PYPI_PACKAGES`, or update the installer manifest so the two agree.

--- a/doc/sphinxext/related_software.py
+++ b/doc/sphinxext/related_software.py
@@ -38,7 +38,6 @@ PYPI_PACKAGES = {
     "meggie",
     "niseq",
     "sesameeg",
-    "mne-kit-gui",  # moved to its own env in the installers
     "zuna",
 }
 

